apps/docs/src/guide/transactions/index.md

```diff
@@ -0,0 +1,14 @@
+# Transactions
+
+A transaction is a way of interacting with Fuel Network's blockchain and can include actions like transferring assets, deploying contracts and minting tokens.
```

Comment:
Feel like we should prefer using this wording over '_the_ xyz blockchain' because we are a VM-focused company, and not an L2 necessarily. 

```suggestion
A transaction is a way of interacting with a Fuel blockchain and can include actions like transferring assets, deploying contracts and minting tokens.
```

Reply:
Our SDK is supposed to work for _any_ blockchain using the FuelVM and not just some one chain that Fuel would deploy.